.simplecov

```diff
@@ -6,5 +6,5 @@ if ENV['CI'] == 'true'
   require 'codecov'
   SimpleCov.formatter = SimpleCov::Formatter::Codecov
 else
-  SimpleCov.formatter = SimpleCov::Formatter::QualityFormatter
+  SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter
```

Comment:
This change throws an error at the end of my test suite: `simplecov-0.16.1/lib/simplecov/result.rb:48:in `format!': private method `format' called for #<Class:0x00007fa1cd8c5ae8> (NoMethodError)`. We may want this to be `HTMLFormatter` instead.